x11-terms/ghostty-terminfo/ghostty-terminfo-1.1.0-r1.ebuild

```diff
@@ -59,16 +59,27 @@ S="${WORKDIR}/${P/-terminfo/}"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64"
+# Tests are run on the main Ghostty package.
+RESTRICT="test"
 
-RDEPEND="!>=sys-libs/ncurses-6.5_p20250118[-minimal]"
+IUSE="symlink"
+RDEPEND="
```

Comment:
I think you need a blocker here on some ncurses as well to allow the file transition between ncurses and ghostty-terminfo.

Reply:
In my tests the file transition was correctly handled by Portage without the blocker. What kind of blocker are you looking for?

Reply:
The latter I understand, but isn't the former handled by the `symlink?` and `!symlink?` clauses?

Reply:
Okay, now that I've tested it in a chroot, I understand why. Added.

Reply:
Let me know when pushed.

Reply:
Done.